src/java/org/apache/cassandra/db/compaction/CompactionStrategyManager.java

```diff
@@ -1039,6 +1022,14 @@ public String getName()
 
     public List<List<AbstractCompactionStrategy>> getStrategies()
     {
+        return getStrategies(true);
+    }
+
+    private List<List<AbstractCompactionStrategy>> getStrategies(boolean checkBoundaries)
+    {
+        if (checkBoundaries)
+            maybeReloadDiskBoundaries();
```

Comment:
This call repeats below. Is this intentional?

Reply:
Fixed in db135b8.